src/App.js

```diff
@@ -1,7 +1,22 @@
 import React from 'react'
+import { Footer } from './component/footer/Footer'
```

Comment:
Since you are creating a folder for each component, you can import components by specifying only the folder name as `./component/footer` if you rename your component files as `index.js`.